packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -502,17 +535,23 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
   }
 
   _initialPositionChanged(initialPosition) {
+    if (this._monthScroller && this._yearScroller) {
+      this._monthScroller.active = true;
+      this._yearScroller.active = true;
+    }
```

Comment:
This was previously set in template using `active="[[initialPosition]]"` to trigger rendering of the scrollers.
We could consider refactoring this to use `afterNextRender()` instead of using the observer.